paypro-gateways-woocommerce/includes/paypro/wc/gateway.php

```diff
@@ -1,18 +1,20 @@
 <?php if(!defined('ABSPATH')) exit; // Exit if accessed directly
 
-abstract class PayPro_WC_Gateway_Abstract extends WC_Payment_Gateway
+class PayPro_WC_Gateway extends WC_Payment_Gateway
 {
-    protected $default_title;
-
-    protected $default_description;
-
-    protected $default_logo;
+	/**
+	 * panther restapi responses
+	 */
+	protected const PAYPROAPIRES_NOTSUBSCRIPTED = "Not subscribed to money transfer service";
```

Comment:
The indentation is different with the lines below

Reply:
Could be make it `PAYPRO_API_RES_NOT_SUBSCRIPTED` for more readable? Also I am thinking about `SUBSCRIBED` instead of `SUBSCRIPTED`

Reply:
updated, also updated it for `apikey` invalid in `PayproAPIHelper`